src/Altinn.App.PlatformServices/Altinn.App.PlatformServices.csproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <OutputType>Library</OutputType>
-    <Version>5.0.0</Version>
-    <AssemblyVersion>5.0.0.0</AssemblyVersion>
+    <Version>5.1.0</Version>
```

Comment:
5.1.0-alpha.1 ref comment above

Reply:
And btw, the last project Altinn.App.Common should be bumped as well although not necessary it makes it easier to be sure we have the correct assemblies.